src/RA_Dlg_AchievementsReporter.h

```diff
@@ -8,34 +8,38 @@ class Achievement;
 
 class Dlg_AchievementsReporter
 {
-public:
-    static const int MAX_ACHIEVEMENTS = 200;
-    static const int MAX_TEXT_LEN = 250;
+    inline static constexpr auto MAX_ACHIEVEMENTS{ 200 };
+    inline static constexpr auto MAX_TEXT_LEN{ 250 };
 
-public:
-    enum ReporterColumns
+    enum class Column : std::size_t
     {
         Checked,
-        //ID,
         Title,
         Desc,
         Author,
-        Achieved,
-
-        NumReporterColumns
+        Achieved
+    };
+    inline static constexpr std::array<LPCTSTR, 5> COL_TITLE
+    {
+        _T(""),
+        _T("Title"),
+        _T("Description"),
+        _T("Author"),
+        _T("Achieved?")
     };
+    inline static constexpr std::array<int, 5> COL_SIZE{ 19, 105, 205, 75, 62 };
 
 public:
     static void DoModalDialog(HINSTANCE hInst, HWND hParent);
     static INT_PTR CALLBACK AchievementsReporterProc(HWND, UINT, WPARAM, _UNUSED LPARAM);
 
 public:
     static void SetupColumns(HWND hList);
-    static int AddAchievementToListBox(HWND hList, const Achievement* pAch);
+    _NODISCARD static int AddAchievementToListBox(_In_ HWND hList, _In_ const Achievement* const pAch);
```

Comment:
I find it humorous that you make this `_NODISCARD` and add an `_UNUSED` variable in the one place that calls it.

Recommend just changing the function to return `void`.

Reply:
alright